content/en/profiler/profiler_troubleshooting/ddprof.md

```diff
@@ -43,10 +42,14 @@ There are two capabilities you can use to override the value of `perf_event_para
 - `CAP_PERFMON`: adds BPF and `perf_event_open` capabilities (available on Linux v5.8 or later)
```

Comment:
```suggestion
- `CAP_PERFMON`: Adds BPF and `perf_event_open` capabilities (available on Linux v5.8 or later).
```

Reply:
I have the same feedback for the line above this one for CAP_SYS_ADMIN (adding a period and capitalizing the first letter of the word "Add" in the description), but I'm unable to add it as a suggested change since it's outside the scope of the PR (i.e., Github won't let me do it).